src/Symfony/Component/HttpClient/Response/CommonResponseTrait.php

```diff
@@ -35,7 +35,7 @@ trait CommonResponseTrait
     private int $offset = 0;
     private ?array $jsonData = null;
 
-    public function getContent(bool $throw = true): string
+    final public function getContent(bool $throw = true): string
```

Comment:
this break tests :)
but honestly I'm not convinced this is usefull.
I'd vote for removing the keyword here and from toStream.